packages/core/src/client/decorators/bundlerClient.ts

```diff
@@ -8,7 +8,7 @@ import type {
   Transport,
 } from "viem";
 import { estimateUserOperationGas } from "../../actions/bundler/estimateUserOperationGas.js";
-import { getSupportedEntryPoints } from "../../actions/bundler/getSupportedEntrypoints.js";
+import { getSupportedEntryPoints } from "../../actions/bundler/getSupportedEntryPoints.js";
```

Comment:
did you mean to change this?

Reply:
i think the builds fails on this pr bc of this

Reply:
huh git did not pick up the file renamed. fixed and pushed again.